CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+# Contributing
+
+Thanks for considering making a contribution to `mongodb`!
+
+## Getting started
+
+1. Clone this repository
+2. Install mongodb and make sure it's available on `$PATH`
+3. Run `start_mongo.bash`
```

Comment:
Might be worth noting that there is also `start_mongo.bat` for windows users.

Reply:
Hmmm.. I don't see one.  Maybe there was in the past perhaps?

The contributing guide I've provided is hardly exhaustive, just meant to be a quick and dirty "get you going" for now.  In reality I've found you need to at least scan over `start_mongo.bash` and the tests to figure out what's going on.